domains/leo.localplayer.dev.json

```diff
@@ -9,10 +9,22 @@
   },
 
   "record": {
-    "MX": ["mx1.simplelogin.co.","mx2.simplelogin.co." ]
-    "SPF": {
-      "TXT": "v=spf1 include:simplelogin.co ~all"
-    },
+    "MX": [
+      {
+        "priority": 10,
```

Comment:
We do not support priority properties for these records.

Reply:
Updated